webpack/assets/javascripts/react_app/components/common/charts/BarChart/BarChart.stories.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 import BarChart from './';
-import { barChartData } from './BarChart.fixtures';
```

Comment:
doesn't it make sense to reuse fixtures data?

Reply:
Read the description

> Instead of importing fixtures use hard-coded props so they would be visible when clicking Show Source

I'm opening it up for a discussion, what is the right approach for stories?

Reply:
sorry, I didn't see it ... doesn't clear from the commit message
I think when the data is a big structure it would be a bit annoying, while I do understand the value from `show source`